.github/workflows/build.yml

```diff
@@ -20,6 +21,7 @@ on:
 jobs:
   universal:
     name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    if: ${{ github.event.pull_request.draft == false }}
```

Comment:
This will make it broken with push event

Reply:
Ruled out the event pull.